test/MerkleLib.Fixture.ts

```diff
@@ -0,0 +1,17 @@
+import { getContractFactory } from "./utils";
+import hre from "hardhat";
+
+export const merkleLibFixture = hre.deployments.createFixture(async ({ deployments }) => {
+  await deployments.fixture();
+  const [signer] = await hre.ethers.getSigners();
+  const merkleLib = await (await getContractFactory("MerkleLib", signer)).deploy();
+  const merkleLibTest = await (
+    await hre.ethers.getContractFactory("MerkleLibTest", {
+      signer,
+      libraries: {
```

Comment:
can this be linted to fewer lines?

Reply:
Are you suggesting changing the prettier settings?

Reply:
just seems like it can be rolled into 
```javascript
...
signer,
libraries: { MerkleLib: merkleLib.address }
```

one one line.